src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTab.java

```diff
@@ -307,6 +307,10 @@ public void setTargetBundle(EntityBundle bundle) {
 		view.clearActionMenuContainer();
 		//Preview
 		view.setPreviewVisible(isFile);
+		
+		fileTitleBar.clearState();
+		folderTitleBar.clearState();
```

Comment:
I would like to make sure that I understand the lifecycle enough to make sure that state is always cleared, otherwise you may have a previous file or folder's search schema. The alternative is to completely clear out both title bars, then only re-add them when you want a certain titlebar.